rosbag2_storage_default_plugins/test/rosbag2_storage_default_plugins/sqlite/test_sqlite_storage.cpp

```diff
@@ -79,19 +79,21 @@ TEST_F(StorageTestFixture, get_next_returns_messages_in_timestamp_order) {
   EXPECT_FALSE(readable_storage->has_next());
 }
 
```

Comment:
this test depends on ``gmock`` but that is now called ``gtest``, right?
https://github.com/google/googlemock

Reply:
I don't understand the question, what exactly do you mean by "depends on `gmock`"?

Reply:
the top of the file includes gmock, not gtest. why? same for the ament rule... i guess i don't understand what gmock does differently than gtest, but i see in the README it helps tests classes better so maybe disregard my comment

Reply:
There are feature that are only accessible when using the gmock header. I don't know if there are plans to unify that on the upstream side. For now I think this has to be as it is.